indexer/services/comlink/__tests__/controllers/api/v4/historical-pnl-controller.test.ts

```diff
@@ -207,5 +207,65 @@ describe('pnlTicks-controller#V4', () => {
         ],
       });
     });
+
+    it('Get /historical-pnl/parentSubaccountNumber', async () => {
```

Comment:
can you create a test to verify that this returns an aggregated pnl of child and parent subaccount?

Reply:
That is exactly what this test does